apps/sms/views/conversation/js/conversation.js

```diff
@@ -407,6 +407,7 @@ var ConversationView = {
   },
 
   getIdIterator: function conv_getIdIterator() {
+    // TODO: Will need to replace with Conversation service method.
```

Comment:
not sure; I think it makes sense that the view itself keeps a local JS model once it's loaded